content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -25,128 +25,120 @@
       </ul>
     </nav>
     <main>
-      <h1>Radio Group Example Using aria-activedescendant</h1>
+      <h1>aria-activedescendant を使用したラジオグループの例</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/7fb7c8e28b55a2804bda682c36945a1154d1096a -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <p>
-          This example implements the features of the <a href="../radio-group-pattern.html">Radio Group Pattern</a> for two radio groups -- one for choosing a pizza crust and another for choosing a delivery method.
-          This implementation uses <code>aria-activedescendant</code> for informing assistive technologies which radio button has visual focus.
+          この例では、<a href="../radio-group-pattern.html">ラジオグループ パターン</a>の機能を、二つのラジオグループのために実装しています -- ひとつはピザのクラストを選ぶためで、もうひとつは受け渡しの方法を選ぶためです。この実装では、どのラジオボタンが視覚的にフォーカスされているかを支援技術に伝えるために <code>aria-activedescendant</code> を使用しています。
```

Comment:
この実装例における visual focus は、「DOMフォーカスではないけれど aria-activedescendant 属性を通じて視覚的にフォーカス状態になっているもの」を指す一連の単語になっているように思いました。これを「視覚的なフォーカス」としてみるのはどうでしょうか？
```suggestion
          この例では、<a href="../radio-group-pattern.html">ラジオグループ パターン</a>の機能を、二つのラジオグループのために実装しています -- ひとつはピザのクラストを選ぶためで、もうひとつは受け渡しの方法を選ぶためです。この実装では、どのラジオボタンが視覚的なフォーカスを持つかを支援技術に伝えるために <code>aria-activedescendant</code> を使用しています。
```

Reply:
ああ、なるほどです。むしろ、素直な訳になって、よいと思います！